ffi/diplomat/src/decimal.rs

```diff
@@ -35,39 +35,17 @@ pub mod ffi {
         Min2,
     }
 
-    pub enum ICU4XFixedDecimalSignDisplay {
-        Auto,
-        Never,
-        Always,
-        ExceptZero,
-        Negative,
-    }
-
-    pub struct ICU4XFixedDecimalFormatOptions {
-        pub grouping_strategy: ICU4XFixedDecimalGroupingStrategy,
-        pub sign_display: ICU4XFixedDecimalSignDisplay,
-    }
-
-    impl ICU4XFixedDecimalFormatOptions {
-        pub fn default() -> ICU4XFixedDecimalFormatOptions {
-            ICU4XFixedDecimalFormatOptions {
-                grouping_strategy: ICU4XFixedDecimalGroupingStrategy::Auto,
-                sign_display: ICU4XFixedDecimalSignDisplay::Auto,
-            }
-        }
-    }
-
     impl ICU4XFixedDecimalFormat {
         /// Creates a new [`ICU4XFixedDecimalFormat`] from locale data.
         #[diplomat::rust_link(icu::decimal::FixedDecimalFormat::try_new, FnInStruct)]
         pub fn try_new(
```

Comment:
(Not going to do it in this pull request, if only to avoid invalidating your reviews faster than you can do them :-p)
Would the plan then be to make `try_new` the eventual name of the function that takes the options struct of the most recent major version once that becomes a possibility?
Or just never to have such an options struct through FFI?

Reply:
> Would the plan then be to make try_new the eventual name of the function that takes the options struct of the most recent major version once that becomes a possibility?

Yeah, that's what I was thinking. We will discuss further in #2136